### ISSUE TITLE
thumbs and previews do not work if user is not logged in on shared public links

### ISSUE THREAD

**Issue by madsi1m** [owncloud/gallery#776]
### Steps to reproduce

we've noticed that the gallery/preview app doesn't work correctly with shared public links in the situation of:
random person opening link looking at pictures does not get thumbs and get the download dialog when clicking the image.
But:
if a user who is already logged in and they go to the shared public link the gallery and thumbs work as expected.

### Expected behaviour
Expect it to work in both situations

### Actual behaviour
Only works if user is logged in

**Comment by phil-davis** [comment]
Similar to #774 ?

**Comment by madsi1m** [comment]
similar yes, but not exactly. I think resolving for one will most probably fix for both. In this case we are not seeing in the file view (in our install we have hidden the gallery button via css)

**Comment by phil-davis** [comment]
Yes, the other issue goes along further adding the public link to someone's cloud...
The steps to reproduce here are shorter. As you say, resolve this issue and likely the other will also be resolved.

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
